tests/FsToolkit.ErrorHandling.TaskResult.Tests/TaskResult.fs

```diff
@@ -823,3 +823,19 @@ let TaskResultOperatorTests =
             )
             |> Expect.hasTaskOkValueSync (PostId newPostId)
     ]
+
+[<Tests>]
+let TaskResultBindRequireTests =
+    testList "TeskResult Bind + Require Tests" [
+        testCase "bindRequireNone"
```

Comment:
Should make these `testCaseTask` as the expect statement you'd want to `return!`/`do!` to ensure it actually runs the expect code rather than fire & forget.

Reply:
Fixed, thank you!